cmd/generate/config/rules/openshift.go

```diff
@@ -0,0 +1,37 @@
+package rules
+
+import (
+	"regexp"
+
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/secrets"
+	"github.com/zricethezav/gitleaks/v8/config"
+)
+
+// OpenShift 4 user tokens are prefixed with `sha256~`.
+// https://docs.redhat.com/en/documentation/openshift_container_platform/4.10/html-single/authentication_and_authorization/index#oauth-view-details-tokens_managing-oauth-access-tokens
+func OpenshiftUserToken() *config.Rule {
+	r := config.Rule{
+		RuleID:      "openshift-user-token",
+		Description: "Found an OpenShift user token, potentially compromising an OpenShift/Kubernetes cluster.",
+		// TODO: Do tokens vary in length or are they always 43?
+		Regex:       regexp.MustCompile(`\b(sha256~[\w-]{43})(?:[^\w-]|\z)`),
+		Entropy:     3.5,
+		SecretGroup: 0,
```

Comment:
It's been a while: I don't remember if `SecretGroup: 0` is necessary.

Either way, `SecretGroup` doesn't seem to be included in the generated config.

Reply:
@rgmz nope, secret group is no longer necessary. The picked capture group will always default to the first capture group.